recipes/atlas-schema/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "atlas-schema" %}
+{% set version = "0.1.0" %}
+{% set python_min = "3.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/atlas_schema-{{ version }}.tar.gz
+  sha256: 50f8fd29e958f00ff5fe8b0f2235e281952e5391e893bb38e61c2c50751b7766
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - pip
+    - hatchling >=0.7
+    - hatch-vcs >=0.3.0
+  run:
+    - python >={{ python_min }}
+    - coffea >=2024.10.0
+    - distributed >=2024.3.0
+    # below are for coffea[dask]
+    - distributed >= 2024.3.0
```

```suggestion
    - distributed >=2024.3.0
```

@kratsg, you typoed the spacing